### ISSUE TITLE
Combobox in desktop Safari hard to get to with VoiceOver

### ISSUE THREAD

**Issue by Greenie10** [reach/reach-ui#645]
### Current Behavior

Navigating with VoiceOver to the "Place of birth" field on [Findmypast sign up form](https://www.findmypast.co.uk/family-tree), trying to get into the input field is hard, have to VO + shift + down arrow to "enter" the combobox, then you still can't type. If you left arrow, the cursor becomes available to type.

### Expected behavior

On navigating to the field, should be able to just start typing (NB: this behaviour works fine on desktop Chrome).

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Reach Package    |    Combobox     | ^0.8.6      |
| React            |   react-dom      |    16.13.1     |
| Browser          |     Safari    |     13.1    |
| Assistive tech   |   VoiceOver      |    macOS Catalina 10.15.4     |
| Node             |         |         |
| npm/yarn         |         |         |
| Operating System |     macOS     |    Catalina 10.15.4     |
